specification/kubernetesconfiguration/resource-manager/Microsoft.KubernetesConfiguration/preview/2021-05-01-preview/examples/LocationExtensionTypes_List.json

```diff
@@ -0,0 +1,114 @@
+{
+  "parameters": {
+    "subscriptionId": "00000000-0000-0000-0000-000000000000",
+    "clusterRp": "Microsoft.Kubernetes",
+    "clusterType": "connectedClusters",
+    "location": "eastus",
+    "api-version": "2021-05-01-preview"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/subscriptions/00000000-0000-0000-0000-000000000000/providers/connectedClusters/locations/eastus/extensionTypes",
```

Comment:
not a valid id .. please fix in all files

Reply:
does it have to be a valid subscription or just a valid guid? 
sorry, I thought this was intended to be a sort of placeholder

Reply:
the example files are generated by `oav generate-examples`, I'll review and update `extensionTypes.json`